build.gradle

```diff
@@ -39,6 +39,21 @@ subprojects {
 	apply from: "${rootDir}/gradle/xtend-compiler-settings.gradle"
 	apply from: "${rootDir}/gradle/maven-deployment.gradle"
 	apply from: "${rootDir}/gradle/manifest-gen.gradle"
+
+	configurations.all {
+		if (name.contains("Tooling")) {
+			resolutionStrategy { 
+				eachDependency { DependencyResolveDetails details ->
+					if (details.requested.group == 'org.eclipse.platform' && details.requested.name == 'org.eclipse.core.runtime') {
+						details.useVersion "3.19.0"
```

Comment:
Is the syntax difference here to line 51 `(` and `)` important.

Reply:
nope they are not, will harmonize